newsletters.xml

```diff
@@ -0,0 +1,20 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8" ?>
+<rss version="2.0">
+<channel>
+  <title>Browser Architecture</title>
```

Comment:
Should this feed say more (Mozilla? Firefox?) for consumers _outside_ of `planet.mozilla.org`?

Reply:
Good point. Probably yes

Reply:
Addressed in 23d494e38fb269584d6eab2248328d8e4a3cf5b3